index.html

```diff
@@ -42,6 +42,56 @@ <h1 style="color: white; font: 500 30px/36.6px Google Sans"> Your Google Ads jou
             </div>
         </div>
 
+        <div class="slide" style="background-color: #f0e9e9;">
```

Comment:
Nit: We have a single white frame with dark text in the middle of dark frames. Can we change this to a dark overlay calendar/clicks image/video (eg. https://www.pexels.com/video/person-using-a-cellphone-6974194/) with white text?